static/ckeditor/ckeditor/plugins/panel/plugin.js

```
@@ -0,0 +1,469 @@
+				if ( !this.onHide || this.onHide.call( this ) !== true )
+					this.element.setStyle( 'display', 'none' );
+			},
+
+			onKeyDown: function( keystroke, noCycle ) {
+				var keyAction = this.keys[ keystroke ];
+				switch ( keyAction ) {
+					// Move forward.
+					case 'next':
+						var index = this._.focusIndex,
+							focusables = this._.getItems(),
+							focusable;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)